jenkins-integration/jenkins-jobs/scenarios/oss-puppetserver-latest-json/JobDSL.groovy

```diff
@@ -0,0 +1,14 @@
+// NOTE: we only want to cron this job and use a 'real' SUT hostname if the job
+// is running on a production server.
+if (serverConfig["environment"] == "production") {
+    job.with {
+        triggers {
+            // This should run the job at a semi-random time between 9:00 and 10:59PM,
+            //  on Mondays.
+            cron('H H(21-22) * * 1')
```

Comment:
We probably don't want this to run at this time and with sut54, right, since this could conflict with the [existing oss-puppetserver-latest job](https://github.com/puppetlabs/gatling-puppet-load-test/blob/e5bad173cd35340c04debee485e5d367eb6fdf48/jenkins-integration/jenkins-jobs/scenarios/oss-puppetserver-latest/JobDSL.groovy#L8-L12)?